test/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "mintlayer-test"
+version = "0.1.0"
+edition = "2021"
+homepage = "https://github.com/mintlayer/core/issues"
```

Comment:
Should the homepage be issues?

Reply:
This is only for the `test` crate. It's presented when tests fail, and is easy to get from Cargo into the system. Could be hard-coded in the code instead though. Would that be preferable?

Reply:
Actually that's a fair point, it's probably a good place to throw at people. I'm fine with this

Reply:
I think the URL should be updated to point to this repository

Reply:
@altonen So the homepage would be this repo, and the url to point people to report bugs would be hardcoded in code or also this repo? I thought it was quite appropriate for homepage for tests to be the issues page but admittedly it's probably not how the homepage key was intended to be used.

Reply:
I'm not sure whether there's a misunderstanding going on. But I think Aaro means to use `mintlayer-core` instead of `core`.

Reply:
Right, thanks for pointing that out!